client/src/components/SolutionViewer/SolutionViewer.tsx

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+import { Panel } from '@freecodecamp/react-bootstrap';
+import Prism from 'prismjs';
+import React from 'react';
+import { ChallengeFile, ChallengeFiles } from '../../redux/prop-types';
+
+type SolutionViewerProps = {
+  challengeFiles: ChallengeFiles;
+  solution: string | null | undefined;
```

Comment:
```suggestion
  solution: string;
```
This one should stay a string - when you handle the fix in the other comment, the error for this being possibly null/undefined will go away.